tests/unit/users/usersService.test.ts

```diff
@@ -321,4 +321,47 @@ describe('Users Service', () => {
 
     expect(result.minimumLength).toBeDefined();
   });
+
+  it('Should allow a user with permissions to view the email template configurations', async () => {
+    // The user service will respond in snake case and the SDK will convert the response to camel case
+    const data = {
```

Comment:
T'is intentional to only provide three fields instead of four as the request / response is technically partial.